components/brave_ads/core/internal/legacy_migration/database/database_migration_issue_17231_unittest.cc

```diff
@@ -21,8 +21,13 @@ namespace brave_ads {
 class BraveAdsDatabaseMigrationIssue17231Test : public UnitTestBase {
  protected:
   void SetUpMocks() override {
-    ASSERT_TRUE(CopyFileFromTestPathToTempPath("database_issue_17231.sqlite",
-                                               kDatabaseFilename));
+    const std::string database_path =
+        base::FilePath()
+            .AppendASCII("database")
+            .AppendASCII("database_issue_17231.sqlite")
+            .AsUTF8Unsafe();
+    ASSERT_TRUE(
+        CopyFileFromTestPathToTempPath(database_path, kDatabaseFilename));
```

Comment:
Maybe add to a database_migration_unittest_util as same code is in 3 places?

Reply:
changed this code back to pass path as a single string